doc/source/licenses-actions/index.rst

```diff
@@ -0,0 +1,43 @@
+Licenses actions
+================
+
+The ``pyansys/check-licenses`` action allows to verify that the project's dependencies
+only contain valid licenses.
+
+The valid licenses are defined in the ``accepted-licenses.txt``. Moreover, an additional txt
+file ``ignored-packages.txt`` contains a list of packages that are trusted but may not have a
+valid license associated.
+
+Use of the check-licenses action
```

Comment:
We are using one section per action. In addition, the title of the section is the title of the action, see the rest of the documentation files. Name this section to something like:

```suggestion
Check licenses action
```

All the content related with this action should be described below this section. For the guides on how to update the TXT files, take advantage of a `.. note::` directive.

Reply:
Done.